aspnet/SameSite/kbs-samesite.md

```diff
@@ -45,4 +45,4 @@ The following table is for earlier versions of Windows and Windows Server.
 | .NET Framework 4.6, 4.6.1, 4.6.2, 4.7, 4.7.1, 4.7.2 | [Catalog](https://www.catalog.update.microsoft.com/Search.aspx?q=4533012)  | [4533012](https://support.microsoft.com/en-us/help/4533012) |
 | .NET Framework 4.8  | [Catalog](https://www.catalog.update.microsoft.com/Search.aspx?q=4533005)  | [4533005](https://support.microsoft.com/en-us/help/4533005) |
 | **Windows Server 2008** | |
-| .NET Framework 4.6  | [Catalog](https://www.catalog.update.microsoft.com/Search.aspx?q=4533012)  | [4533012](https://support.microsoft.com/en-us/help/4533012) |
\ No newline at end of file
+| .NET Framework 4.6  | [Catalog](https://www.catalog.update.microsoft.com/Search.aspx?q=4533012)  | [4533012](https://support.microsoft.com/en-us/help/4533012) |
```

Comment:
There is no change here if I compare the text in the original using a different tool.  Whatever it is registering as diff change here however is not going to be a problem, so I am ignoring it.